src/modules/Hosts/Hosts/Strings/en-us/Resources.resw

```diff
@@ -272,6 +272,17 @@
   <data name="ShowOnlyDuplicates.Header" xml:space="preserve">
     <value>Show only duplicates</value>
   </data>
+  <data name="TooManyHostsTeachingTip.Subtitle" xml:space="preserve">
+    <value>Only 9 hosts per entry are supported</value>
+    <comment>"Hosts" refers to the system hosts file, do not loc</comment>
+  </data>
+  <data name="TooManyHostsTeachingTip.Title" xml:space="preserve">
+    <value>Entries with too many hosts found</value>
+    <comment>"Hosts" refers to the system hosts file, do not loc</comment>
+  </data>
+  <data name="TooManyHostsTeachingTipContent.Text" xml:space="preserve">
+    <value>The affected entries have been splitted. Changes won't take effect until the next saving.</value>
```

Comment:
`Changes won't take effect until the next saving` can be a bit unclear IMHO. I get that the save will happen on next change from Hosts app, but someone else might not get that. Not sure how to phrase it differently though

Reply:
Good catch. Let's tweak the message to `Changes will take effect on next change from the app.` and remove also the negation! 😃 
I'll update the PR.

Reply:
`This will take effect on next entry change.` is more clear.